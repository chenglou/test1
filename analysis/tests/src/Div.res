
let q = <div> </div>
//        ^hov