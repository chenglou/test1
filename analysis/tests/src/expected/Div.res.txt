Hover tests/src/Div.res 1:10
{"contents": "```rescript\n(\n  string,\n  ~?props: ReactDOMRe.domProps,\n  array<React.element>,\n) => React.element\n```"}

Complete tests/src/Div.res 3:3
[{
    "label": "aria-details",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "aria-disabled",
    "kind": 4,
    "tags": [],
    "detail": "bool",
    "documentation": null
  }, {
    "label": "key",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }]

